roles/satellite-clone/tasks/reset_pulpcore_data.yml

```
@@ -0,0 +1,22 @@
+- name: reset pulpcore data
+  command: echo "Katello::Rpm.all.destroy_all; Katello::Erratum.all.destroy_all; Katello::PackageGroup.all.destroy_all; Katello::DockerManifest.all.destroy_all; Katello::DockerTag.all.destroy_all; Katello::AnsibleCollection.all.destroy_all" | foreman-rake console
+
+- name: reset 7.0+ specific pulpcore data
+  command: echo "Katello::GenericContentUnit.all.destroy_all" | foreman-rake console
+  when:
+    - satellite_version in ["7.0"]
```

If it's 7.0+, should this be:
```suggestion
    - satellite_version not in ["6.10"]
```

---

indeed.